build.js

```diff
@@ -344,7 +344,24 @@ const flattenIDL = (specIDLs, customIDLs) => {
     (dfn) => dfn.type !== 'includes' && dfn.type !== 'interface mixin'
   );
 
-  return {ast, globals};
+  // Get all possible scopes
+  const scopes = new Set();
+  for (const dfn of ast) {
+    // Special case RTCIdentityProviderGlobalScope since it doesn't use the
+    // Global extended attribute correctly:
+    // https://github.com/w3c/webrtc-identity/pull/36
+    if (dfn.name === 'RTCIdentityProviderGlobalScope') {
+      scopes.add('RTCIdentityProvider');
+      continue;
+    }
+
+    const attr = getExtAttrSet(dfn, 'Global', false);
+    for (const s of attr) {
+      scopes.add(s);
+    }
```

Comment:
To avoid the `throwError` argument, we can make this code just a little bit more complicated:

```suggestion
    const attr = getExtAttrSet(dfn, 'Global');
    if (attr) {
      for (const s of attr) {
        scopes.add(s);
      }
    }
```

It should be a simplification overall.